src/app/app.component.ts

```diff
@@ -170,6 +182,18 @@ export class AppComponent implements OnDestroy, OnInit {
             }
         });
 
+        this.policyListService.addPolicies([
+            new TwoFactorAuthenticationPolicy(),
+            new MasterPasswordPolicy(),
+            new PasswordGeneratorPolicy(),
+            new SingleOrgPolicy(),
+            new RequireSsoPolicy(),
+            new PersonalOwnershipPolicy(),
+            new DisableSendPolicy(),
+            new SendOptionsPolicy(),
+            new ResetPasswordPolicy(),
+        ]);
+
```

Comment:
I know you weren't happy about loading policies here, but I think it's perfect besides being in app.component. We get clear control and vision on what policies are supported & the addPolicies call taking an array is pretty clean.

Just curious, could we not populate the service in policies.component? What about in a constructor for the policy list service?